dev_guide/cron_jobs.adoc

```diff
@@ -46,7 +46,9 @@ endif::[]
 ====
 A cron job creates a job object approximately once per execution time of its
 schedule, but there are circumstances in which it will fail to create a job or
-two jobs might be created. As a result, jobs should be idempotent.
+two jobs might be created. As a result, jobs should be idempotent.  For this reason
```

Comment:
s/As a result ... frequently/`Therefore, jobs should be idempotent and you should xref:#cleaning-up-after-a-cron-job[configure history limits]
frequently.`

Reply:
Remember no `#` when using `xref:...`